src/components/tab-bar/tab-bar.tsx

```diff
@@ -87,9 +94,11 @@ export class TabBar {
         };
 
         this.setupListeners();
+        this.handleScroll();
```

Comment:
Is it clear enough why we'd want to do this here, or should I add a comment? (This is in the `setup` function. Expand the code to see the context.)

Reply:
I think its clear enough :)